### DISCUSSION TITLE
-bash: /etc/wiki/.db-secret: Permission denied

### DISCUSSION THREAD

**Discussion by MDouglasBTS** [requarks/wiki#7196] [Guides]
Hi,

Apologies if this is trivial but I'm having trouble setting up wiki.js.  When i run the following command - sudo openssl rand -base64 32 > /etc/wiki/.db-secret - I get this error:

-bash: /etc/wiki/.db-secret: Permission denied

I'm not aware of what I'm doing wrong or what permission is not allowed.  Any thoughts?

**Comment by jhm-ciberman** [comment]
Probably is a permissions error:

```bash
sudo chown $USER:$GROUP /etc/wiki/.db-secret
openssl rand -base64 32 > /etc/wiki/.db-secret
```

**Reply by FIorian** [reply]
note for future visitors: I had to create the file first before I could set these permissions:
```
sudo touch /etc/wiki/.db-secret
```
